ui/src/pages/occurrences/occurrences.tsx

```diff
@@ -72,6 +73,7 @@ export const Occurrences = () => {
     occurrences?.some((occurrence) => occurrence.id === id)
   )
   const { selectedView, setSelectedView } = useSelectedView('table')
+  const { taxaLists } = useTaxaLists({ projectId: projectId as string })
```

Comment:
We are fetching taxa lists from both this view and from the control itself. I see why, but let's think about how we can avoid this... I will get back to you! :)

Reply:
I would love to hear more from you about this multi-fetch problem!